src/main/java/de/tum/in/www1/artemis/service/hestia/structural/StructuralAttribute.java

```diff
@@ -0,0 +1,47 @@
+package de.tum.in.www1.artemis.service.hestia.structural;
+
+import java.util.Arrays;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
```

Comment:
Maybe a short description for what these classes are used (all Structural classes)?